lib/Request.js

```diff
@@ -88,6 +89,24 @@ Request.prototype.getParam = function(param) {
     return this._params.hasOwnProperty(param) ? this._params[param] : undefined;
 };
 
+Request.prototype.setQueries = function(queries) {
+    this._url.query = queries;
+    return this;
+};
+
+Request.prototype.getQueries = function() {
+    return this._url.query;
+};
+
+Request.prototype.setQuery = function(query, value) {
+    this._url.query[query] = value;
+    return this;
+};
+
+Request.prototype.getQuery = function(query) {
+    return this._url.query.hasOwnProperty(query) ? this._url.query[query] : undefined;
+};
+
```

Comment:
this is how we currently do it for all properties.... if we do want to use getters and setters we will have to do it for the file... i can put in a ticket to investigate but i'd like to get this in for now so we can start building our end points requiring pagination